src/modals/SharePoll.jsx

```diff
@@ -1,16 +1,103 @@
 import React from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from "styled-components";
+import Button from "../components/Button";
+import ShareIcon from 'assets/ShareIcon.png';
 
+const ModalIconField = styled.div`
+  height: 50px;
+  width: 50px;
+  background: #F5DFBD 0% 0% no-repeat padding-box;
+  border-radius: 50%;
+  display: inline-block;
+  align-self: center;
+`;
+
+const ModalHeader = styled.h5`
+  text-align: center;
+  letter-spacing: 0px;
+  color: #000000;
+  opacity: 1;
+  font-weight: normal;
+  font-size: 30px;
+  padding-top: 20px;
+  padding-bottom: 22px;
```

Comment:
```suggestion
  text-align: center;
  font-weight: normal;
  color: ${({ theme }) => theme.colors.neutral[700]};
  font-size: 30px;
  margin-bottom: 8px;
```
Do odsuwania elementów od siebie stosuje się `margin`. `padding` służy do dodawania wypełnienia do elementów. `opacity: 1` oraz `letter-spacing: 0px` są defaultowo ustawiane takze nie trzeba tego pisać.